### PR TITLE
fix(SAI): Fix Stormwind citizens to be more Blizzlike in their behavior

### LINKED ISSUE #6565: [NPC] Lil Timmy does not despawn
Originally reported: https://github.com/chromiecraft/chromiecraft/issues/979

#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [ ] Generic
- [x] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

Npc [Lil Timmy](https://www.wowhead.com/npc=8666/lil-timmy#comments) doesn't DESPAWNS.
Current state: his path and spawn-despawn point is correct. At spawn-despawn point he is getting IMMEDIATELY spawn after trip around Stormwind.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

According to several versions of wowhead pages(classic, tbc, current, ru-en) and comments there Lil Timmy should act like this:
1) Spawn at the point 47,56
2) Trip around the Stormwind
3) At the point 47,56 despawn
4) Start of internal random spawn timer between 1h-4h

Tester: Most sources from 3.0+ talk about 1:30 hour respawn time
##### SOURCE:

Wowhead page is contain everything what you need to know [Lil Timmy](https://www.wowhead.com/npc=8666/lil-timmy#comments)
##### STEPS TO REPRODUCE THE PROBLEM:

Well, everytime when he is hit spawn-despan point he doesn't disapear and you can see spawn animation. Spot 47,56

##### EXTRA NOTES:

Type here.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/34da0cda5195dede48cb0406b23248330d6249cb

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/99451414-npc-lil-timmy-does-not-despawn?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### LINKED ISSUE #9144: Donna and William models is messed up and under textures path
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

Original report: https://github.com/chromiecraft/chromiecraft/issues/2308

William is chasing Donna.
u0By3D7

Tester:

The issue is confirmed. Seems like text is correct but models is messed up. Also at 59.9, 46.2 they drop down under the bridge or textures. Probably there reset of script but start point in wrong place.

WoWScrnShot_111221_095822

### Expected Blizzlike Behaviour

1. Donna should be shaing William
2. Donna and William should not run under textures

### Source

https://wowpedia.fandom.com/wiki/Donna
https://wowpedia.fandom.com/wiki/William

### Steps to reproduce the problem

.tele stormwind

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/12e92b58082dfda8b73b50aa97b98b9479bb7629

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

### LINKED ISSUE #10060: Defias Prisoner and Stormwind City Guards waypoint incomplete
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [X] 40-49
- [ ] 50-59

### Current Behaviour

Original report: https://github.com/chromiecraft/chromiecraft/issues/1796

WoWScrnShot_090221_134204

Tester:

The issue is confirmed. [Defias Prisoner](https://wowgaming.altervista.org/aowow/?npc=1706) and [Stormwind City Guard](https://wowgaming.altervista.org/aowow/?npc=37063)s has script issues. Disappear animation at Stockade entrance is also weird

WoWScrnShot_010722_201956

### Expected Blizzlike Behaviour

Goup should not be walking on top of eachother

### Source

Wowpedia [Defias Prisoner](https://wowpedia.fandom.com/wiki/Defias_Prisoner?oldid=2364805)
> A Defias Prisoner can also be found at the Stormwind Harbor, escorted by 4 Stormwind City Guards from The Bravery to the Stormwind Stockade. In that case he is friendly and cannot be aggroed or attacked.

### Steps to reproduce the problem

X

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/7e2e6f8ee85f82f57a3c6e0f6f645b4c42292e7a

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by LannyE**
Fixes:
Reginald Windsor is dead. I saw it with my own eyes - Removed
Adam and Billy - Talk, formation, and Waypoints
Removed Extra Guard just outside trade district
Justin, Brandon, and Roman - Talk, formation, and Waypoints
Miss Danna and students - Talk, formation, and waypoints - Add Geoff NPC
Donna and William - Formation and waypoint fix Closes #9144
Janey, Suzanne, and Lisan - Talk, formation, and waypoints
Orphanage/Matron/Shellenne - Waypoints and permanent placement
Karlee, Paige, Gil, and Fizzles - Talk, formation, and waypoints - Add Fizzles creature
Lil' Timmy - Talk, formation, waypoints, and despawn/disappear timer
Closes #6565
Underwater Construction Worker - Removed (gone in WOTLK)
Defias Prisoner and Guards - Formation fix Closes #10060
Sewer Beast - Creature Pool and spawn locations

These changes were made with as much info as I can gather from
wowhead, youtube and other sources.  May not be 100% Blizzlike.
Any existing waypoints were preserved with only slight modifications
due to bridge swallowing children and following NPCs getting stuck.
Some infomation was pulled from the Trinitycore database

This PR does not introduce the day and night cycle there is another PR I have coming with that info as Miss
Danna shouldn't be dragging her students out in the middle of the night to stare at statues and buildings

## Changes Proposed:
-  See description
-  

## Issues Addressed:

- Closes #9144
Closes #6565
Closes #10060

## SOURCE:

Timmy's spawn time is not absolutely known.  I have read everythin from 90 minutes to 3 hours.  I randomized his reappearance between these times.
https://tbc.wowhead.com/npc=8666/lil-timmy#comments
Matron Nightingale is not only part of Children's Week.  She is always there.  Youtube video shows her outside Orphanage.
https://www.youtube.com/watch?v=s4nBCcq9qe4&ab_channel=ChaeriASMR
Also nothing in wowhead says she only spawns during children's week.
Reginald died in The Great Masquerade quest - He is long gone in WOTLK
https://wowwiki-archive.fandom.com/wiki/Reginald_Windsor

## Tests Performed:

- Spent many days staring and following Civilians
- Merged fine in my environment

## How to Test the Changes:

1. Merge
2. Look for the removed NPCs
3. Find the other NPCs and observe their behavior
4. look for Sewer Beast in one of his 6 locations

## Known Issues and TODO List:

- [Timmy doesn't despawn he disappears.  This makes his kitten following him and the random reappearance more reliable - May have to revisit this later.]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Lynethris** [comment]
Tested and all seems good. (issues close check).

Few images:
<details>
WoWScrnShot_032422_181557

WoWScrnShot_032422_181244

</details>

**Comment by Lynethris** [comment]
I had not reset the db, and I found a bug by chance. I have check with and without PR:

With:
WoWScrnShot_032422_203824

Without:
WoWScrnShot_032422_203219

**Comment by LannyE** [comment]
Two words: Reginald Windsor.  The guy was so badass in the previous expansions removing him from the game rips the turrent off the steam tank and makes it lose its texture.  This was a good catch but to be honest this is not my SQL that had the error.  My SQL only uncovered this strange bug in a completely unrelated model.  If I remove Reginald's creature (guid: 86900) from the creature table the tank texture and turrent breaks.   Anyways, long story short I removed that tank modelid 25289 from the steam tank and replaced it with 25341.  Modelid 25289 is not being used anywhere else nor is the refurbished steam tank creature_template used by any other creature entity.  It seems to be working fine now.

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1648089119541675100.sql [hunk 1]
```diff
@@ -0,0 +1,1194 @@
...
+-- -------------------------------------------------Defias Prisoner Escort---------------------------------------------------------------------------------------------------
+
+DELETE FROM smart_scripts WHERE `entryorguid` = 37063;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`event_param5`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_param4`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(37063,0,0,0,38,0,100,0,1,0,0,0,0,29,0,50,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow'),
+(37063,0,1,0,38,0,100,0,2,0,0,0,0,29,0,160,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow'),
+(37063,0,2,0,38,0,100,0,3,0,0,0,0,29,0,200,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow'),
+(37063,0,3,0,38,0,100,0,4,0,0,0,0,29,0,310,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow');
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1648089119541675100.sql:1194]: need a line break at the end.

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Comment by Nyeriah** [comment]
Could you provide sources for this? Because he was added to Stormwind during wrath with the removal of the onyxia chain. Before wrath he could only be found inside the cave. Sources say he was removed in 4.0x https://wowpedia.fandom.com/wiki/Reginald_Windsor

**Comment by LannyE** [comment]
Reginald Windsor died in the quest The Great Masquerade when he was escorted back to Stormwind Keep to expose Onyxia.  I did this quest about 15 times maybe more on retail.  That quest was removed in 3.0.2.  https://wowwiki-archive.fandom.com/wiki/Quest:The_Great_Masquerade  The link from wowpedia does say he is alive in WOW but gives no info as to how or where he is at.  In Azerothcore his NPC is walking the waypoints that he took during the quest which shows it was a leftover or lifted from the quest.  Also the Trinitycore database does not have him as a creature entity anywhere.  I just can't find anything that says he survived somehow.  I dredged up this story from Google cache in 2013 https://webcache.googleusercontent.com/search?q=cache:mx8eAKJewMMJ:https://www.engadget.com/amp/2013-03-20-know-your-lore-marshal-reginald-windsor-and-the-great-masquerad.html+&cd=36&hl=en&ct=clnk&gl=us  and they do not talk about him surviving.  Everything else I find on the internet says he is gone and nothing has shown me a location he could have been at after TBC if he were alive.

**Comment by Lynethris** [comment]
Work

**Comment by acidmanifesto** [comment]
Still need that line break at the end of the sql file as we require all sql files to do so.

**Comment by LannyE** [comment]
GIT and I are not friends.  I thought I had resolved it and my push said I was behind so I thought I was done.  I attempted another push with the space at the end.  Hopefully this worked.

**Comment by LannyE** [comment]
Here is a picture of that guard that is showing in his screenshot from a different angle. 
image

**Comment by acidmanifesto** [comment]
> Here is a picture of that guard that is showing in his screenshot from a different angle. 
> image
> 
> 

Ah ok. Thanks for clarifying that.

**Pull request merged**